apps/src/sites/hourofcode.com/pages/public/index.js

```diff
@@ -162,21 +250,33 @@ function validateFields() {
     }
   }
 
-  if  ($("#hoc-event-type").val() === "out_of_school") {
-    if ($("#organization-name").val() === "") {
-      $('#organization-name-error').show();
+  if (($("#country").val() === 'US') && ($("#hoc-event-type").val() === "in_school")) {
+    if (!schoolData.nces) {
+      schoolData.showDropdownError = true;
       return false;
     } else {
-      $('#organization-name-error').hide();
+      schoolData.showDropdownError = false;
     }
-  }
 
-  if ($("#hoc-event-location").val() === "") {
-    $('#event-location-error').show();
-    return false;
+    if (schoolData.nces === "-1") {
```

Comment:
I think there's a comment in the `SchoolAutocompleteDropdown` component that explains that the value if school isn't found is -1, but might be worth explaining that with a comment here too.  That -1 seems mysterious if you're just looking at it without the full context.

Reply:
I added a SCHOOL_NOT_FOUND const that I use in place of "-1" throughout the code and put a comment at the declaration.